plugin/storage/grpc/proto/storage.proto

```diff
@@ -117,19 +120,19 @@ message FindTraceIDsSuccess {
 
 service SpanWriterPlugin {
     // spanstore/Writer
-    rpc WriteSpan(WriteSpanRequest) returns (google.protobuf.Empty);
+    rpc WriteSpan(WriteSpanRequest) returns (WriteSpanResponse);
 }
 
 service SpanReaderPlugin {
     // spanstore/Reader
-    rpc GetTrace(GetTraceRequest) returns (GetTraceSuccess);
-    rpc GetServices(GetServicesRequest) returns (GetServicesSuccess);
-    rpc GetOperations(GetOperationsRequest) returns (GetOperationsSuccess);
+    rpc GetTrace(GetTraceRequest) returns (GetTraceResponse);
```

Comment:
I suggest implementing this similar to FindTraces, as a stream of chunks, because even a single trace could be larger than the default message limit.

Reply:
Something that I'm unsure about and may need consideration is that having a Trace as a stream of Span means that the ProcessMap and Warnings fields are lost on the Trace.

Reply:
That's true, but these two concepts only apply to the current JSON API used by the UI, they don't apply to the storage API (even though the storage API returns `model.Trace`, it never actually populates ProcessMap or warnings, that's done post-retrieval).